core/src/v2/models.rs

```diff
@@ -221,6 +223,73 @@ pub struct License {
     pub url: Option<String>,
 }
 
+/// Security Definition object.
+///
+/// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#securityDefinitionsObject
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct SecurityDefinition {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub name: Option<SecurityScheme>,
+}
+
+/// Security Scheme object.
+///
+/// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#security-scheme-object
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct SecurityScheme {
+    pub type: String,
```

Comment:
This should be `type_` (as it interferes with Rust keyword) and it should have `#[serde(rename = "type")]`

Reply:
Interesting, so `#[serde(rename = "type")]` chagnes it during de/serialisation?